library/Backends/ArithmeticCircuit/arithmetic-circuit.cabal

```diff
@@ -0,0 +1,91 @@
+cabal-version: 1.12
```

Comment:
this file should not be committed, since we are using stack / hpack

Reply:
ditto for subsequent files

Reply:
my bad, it got through, as I'm often adding files fast